### ISSUE TITLE
Clicking on stable coins (non vouchers) on Pools to show all info

### ISSUE THREAD

**Issue by WillRuddick** [grassrootseconomics/sarafu.network#118]
can we have all the data for cUSD and other tracked vouchers on the voucher page:
seeing their network volume etc?

![image](https://github.com/user-attachments/assets/8bd103bd-1324-4e48-b4a7-4f4622ac7279)

**Comment by williamluke4** [comment]
@kamikazechaser Will we are tracking cUSD transactions with the tracker?

**Comment by kamikazechaser** [comment]
Yeah, we will track cUSD of anyone who has interacted with sarafu.network atleast once.

**Comment by WillRuddick** [comment]
currently showing the wrong decimals
![image](https://github.com/user-attachments/assets/cd59c296-0ac7-4863-91d0-c003de156a30)
